Sources/BraintreePayPal/BTPayPalCheckoutRequest.swift

```diff
@@ -211,11 +215,15 @@ import BraintreeCore
             checkoutParameters["country_code"] = shippingAddressOverride?.countryCodeAlpha2
             checkoutParameters["recipient_name"] = shippingAddressOverride?.recipientName
         }
-        
+
         if let recipientEmail = contactInformation?.recipientEmail {
             checkoutParameters["recipient_email"] = recipientEmail
         }
-        
+
+        if let contactPreference = contactPreference {
```

Comment:
With above changes can do:
```suggestion
        if contactPreference  != .none {
```